src/rtc.rs

```diff
@@ -28,11 +37,12 @@ const LSE_HERTZ: u32 = 32_768;
   [examples/blinky_rtc.rs]: https://github.com/stm32-rs/stm32f1xx-hal/blob/v0.7.0/examples/blinky_rtc.rs
 */
 
-pub struct Rtc {
+pub struct Rtc<CS> {
```

Comment:
Default type could be added
```suggestion
pub struct Rtc<CS=RtcClkLse> {
```